mslib/mscolab/models.py

```diff
@@ -205,7 +205,7 @@ class Change(db.Model):
     commit_hash = db.Column(db.String(255), default=None)
     version_name = db.Column(db.String(255), default=None)
     comment = db.Column(db.String(255), default=None)
-    created_at = db.Column(db.DateTime, default=datetime.datetime.utcnow.replace(microsecond=0))
+    created_at = db.Column(db.DateTime, default=datetime.datetime.utcnow().replace(microsecond=0))
```

Comment:
this sounds not correct.  We need fractions of seconds which are stored in the database.

We have to use them